src/renderer/components/grid-item.jsx

```diff
@@ -8,6 +8,10 @@ import colors from '../utils/colors';
 import styles from '../styles/components/grid-item';
 import GetComponentStyle from './mixins/get-component-style';
 
+export function getFormattedName(name) {
```

Comment:
Why does the method have to be public? Just for testing? I prefer behaviour tests over implementation test.